pkg/bucketeer/api/api.go

```diff
@@ -71,15 +72,33 @@ func (c *client) sendHTTPRequest(url string, body interface{}) ([]byte, error) {
 	}
 	resp, err := client.Do(req)
 	if err != nil {
-		return nil, err
+		return nil, 0, err
 	}
 	defer resp.Body.Close()
 	if resp.StatusCode != http.StatusOK {
-		return nil, NewErrStatus(resp.StatusCode)
+		return nil, 0, NewErrStatus(resp.StatusCode)
 	}
 	data, err := io.ReadAll(resp.Body)
 	if err != nil {
-		return nil, err
+		return nil, 0, err
+	}
+	size, err := c.getBodySize(resp)
+	if err != nil {
+		return nil, 0, err
+	}
+	return data, size, nil
+}
+
+func (*client) getBodySize(resp *http.Response) (int, error) {
+	// Given keys are case insensitive in `func (Header) Get`.
+	// FYI: https://pkg.go.dev/net/http#Header.Get
+	header := resp.Header.Get("Content-Length")
```

Comment:
I think you can use this and simplify it.

Reference: https://pkg.go.dev/net/http#Response
```go
// ContentLength records the length of the associated content.
// The value -1 indicates that the length is unknown.
// Values >= 0 indicate that the given number of bytes may
// be read from Body.
//
// For client requests, a value of 0 with a non-nil Body is
// also treated as unknown.
ContentLength int64
```